src/main/java/org/ilite/frc/robot/Hardware.java

```diff
@@ -64,24 +60,12 @@ public PowerDistributionPanel getPDP() {
     return mPDP;
   }
   
-  public boolean isNavXReady() {
+  public boolean isGyroReady() {
     return mNavxReady.get();
```

Comment:
This method is called isGyroReady and uses the mNavxReady. There's also a method called getGyro and uses the AHRS. Should th